Source/Assimil8or/Assimil8or.cpp

```diff
@@ -667,372 +920,22 @@ void Assimil8orPreset::parse (juce::StringArray presetLines)
         }
 
         LogParsing (juce::String (scopeDepth) + "-" + presetLine.trimStart ());
-        const auto key { presetLine.upToFirstOccurrenceOf (":", false, false).trim () };
-        const auto value { presetLine.fromFirstOccurrenceOf (":", false, false).trim () };
-
-        auto keyIs = [&key] (const juce::String& desiredKey)
-        {
-            return key.upToFirstOccurrenceOf (" ", false, false) == desiredKey;
-        };
-        auto getParameterIndex = [&key] ()
-        {
-            return key.fromFirstOccurrenceOf (" ", false, false).getIntValue ();
-        };
+        key = presetLine.upToFirstOccurrenceOf (":", false, false).trim ();
```

Comment:
We can catch the empty param here, instead of in dispatch,

Reply:
Let me clarify, assuming the empty parameter comes from an actual empty line in the file, let's catch the empty line just before this assigment to key, 

```
if (presetLine.trim ().isEmpty)
    continue;
```